src/utils.ts

```diff
@@ -1,5 +1,13 @@
+import bignum from 'bignum';
+import { ConfigManager } from 'confmgr/lib';
 import log4js from 'log4js';
 
+import { faucetConfig } from './faucetConfig';
+
+const config = faucetConfig('server');
+const decimals = config.Get('BACKEND', 'NETWORK_DECIMALS') as number;
```

Comment:
Add an assertion `assert(Number.isSafeInteger(MAX_ALLOWED_DRIP_FLOAT))` for sanity's sake, otherwise multiplications might become bugged

Reply:
My previous comment was wrong, sorry. I corrected it now.

This was a suggestion to catch errors where the value is wrong for some reason. Try the following:

```
const decimals = 100000000000000000000000000000000000000000000000
Number.isSafeInteger(decimals) // false
```

If you pass too big of a number which can't be represented as an integer, it'll become unsafe. So this is just an extra safety measure to ensure we don't accidentally use an invalid value here. I don't know how big of a value you're expecting for this variable.

Reply:
`decimals` is also constant, (we certainly need to make naming more consistent), so also won't change by user. 
The only things will be user-inputed - is `amount: string`, which (per [conversation](https://github.com/paritytech/substrate-matrix-faucet/pull/150#discussion_r858509240) with @chevdor) could be big enough to produce number which is bigger than safe integer, that's why we use bignum also